src/components/WebexVideoSettings/WebexVideoSettings.jsx

```diff
@@ -16,15 +18,23 @@ import WebexMeetingControl from '../WebexMeetingControl/WebexMeetingControl';
  */
 export default function WebexVideoSettings({meetingID, className, style}) {
   const cssClasses = webexComponentClasses('video-settings', className);
+  const {localVideo} = useMeeting(meetingID);
 
   return (
     <div className={cssClasses} style={style}>
-      <Title>Camera</Title>
-      <WebexMeetingControl type="switch-camera" meetingID={meetingID} />
-      <div className="media">
-        <WebexLocalMedia banner="Preview" mediaType="video" meetingID={meetingID} />
-        <div className="preview"><h3>Preview</h3></div>
-      </div>
+      {localVideo.stream
```

Comment:
i think we should test here that there is no camera in the list. What happens if the user clicks "stop video" before entering settings?